weave-js/src/components/PagePanelComponents/Home/Browse3/filters/CellFilterWrapper.tsx

```diff
@@ -11,6 +11,7 @@ type CellFilterWrapperProps = {
   field: string;
   operation: string | null;
   value: any;
+  style?: React.CSSProperties;
```

Comment:
I think Jamie mentioned we really should be styling things with className in a tailwind context.